brainiak/funcalign/fastsrm.py

```diff
@@ -977,7 +979,7 @@ def fast_srm(reduced_data_list,
         for j in range(n_sessions):
             timeframes_slices.append(
                 slice(current_j, current_j + len(reduced_data_list[0, j])))
-            current_j = len(reduced_data_list[0][j])
+            current_j += len(reduced_data_list[0][j])
```

Comment:
It would be helpful to add to the comments for this function to describe how `reduced_data_list` differs from what SRM needs and hence why you need to reshape it like you do

Reply:
As FastSRM API is more permissive than DetSRM or ProbSRM because the input data can maintain the division by session usually seen in fMRI datasets. This is why we need to concatenate the data across sessions when performing DetSRM. The code enables FastSRM to revert this operation in order to get a consistent API. (I added a comment in the code).